iron-selectable.html

```diff
@@ -132,6 +132,8 @@
     attached: function() {
       this._observer = this._observeItems(this);
       this._contentObserver = this._observeContent(this);
+      this._removeListener(this.activateEvent);
```

Comment:
Right now it's only being added in activateEventChanged, which makes sense because the listener should be updated if the event changes or if it's added / removed from the DOM. We found that in shady DOM in FF, this element got hammered with 2x attached + 2x detached calls (not sure if because of shady DOM or web components polyfill) and we were losing all of the click (tap) events on paper-menu's, so there should be some protection against that sort of thing.